### DISCUSSION TITLE
Ability to enable/disable sensors' "Mode Settings" for Away/Home Mode

### DISCUSSION THREAD

**Discussion by ampersandru** [tsightler/ring-mqtt#649] [Feature Requests]
Is there a way to enable/disable sensors' Mode Settings with Ring-MQTT? This is the setting which lets you ignore or monitor a sensor in specific modes (e.g., away or home).

For example, I want to set up an automation in HA that when both of my dogs are not at home and my Ring alarm is armed, it enables "Away Mode" for the Ring Motion Sensors in the house. If the dogs are at home, and the alarm is armed, make sure that the motion sensor's "Away Mode" is disabled.

The reason I want this is because I've had false positives with the motion sensors that made my alarm go off because of dog motion.

**Comment by tsightler** [comment]
It's not currently possible, but you can use an automation to set the impacted sensors bypass mode to "Always" when the dogs are home, so that the sensors will be bypassed during arming.
